backend/src/services/mempool_monitor.py

```
@@ -1,257 +1,259 @@
+        # Raydium specific: 'Program log: Instruction: Initialize2'
+        elif any("Instruction: Initialize2" in log for log in logs) and any(RAYDIUM_LIQUIDITY_POOL_V4_ID in log for log in logs):
+            is_new_token = True
+
+        # Standard SPL Token: 'Program log: Instruction: InitializeMint'
+        elif any("initializeMint" in log.lower() for log in logs):
+            is_new_token = True
+
+        if is_new_token:
+            logger.info(f"Potential new token/pool transaction detected: {signature}")
+            # Run in background to not block the listener
+            asyncio.create_task(self._process_new_token_transaction(signature))
```

Ruff (0.15.6)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)